laravel/tests/Unit/MedhubControllerTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Tests\Unit;
+
+use Tests\TestCase;
+use App\Http\Controllers\MedhubController;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class MedhubControllerTest extends TestCase
+{
+    use DatabaseTransactions;
+    
+    public function testMedHubAPIConnection()
+    {
+        $MHC = new MedhubController();
+        $testPOST = json_decode($MHC->testPOST()->getBody(), true);
+        $response = $testPOST["response"];
+        $this->assertTrue($response == "success");
+    }
+
+    public function testMedHubActiveResidentsPOST()
+    {
+        $MHC = new MedhubController();
+        $usersArr = json_decode($MHC->activeResidentsPOST()->getBody(), true);
+        $this->assertNotNull($usersArr);
```

Comment:
is it possible to make any stronger (more specific) of an assertion about the responses we will get from Medhub?

For example, will the json data always be an array? Will it always contain certain keys?

If we know it always gives us data of the following form, I believe we ought to confirm that.

```jsonc
{
  "users": [
    {
      "name" : "some string",
      "id": 123123,
      "etc": "something else"
    },
    {...}
  ]
}
```

Reply:
This was addressed, the code was changed.